### PR TITLE
Website Mergebot Job

### PR DESCRIPTION
**Pull request by jasonkuster**
Signed-off-by: Jason Kuster <jasonkuster@google.com>

Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`.
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by jasonkuster** [comment]
R: @tgroh

File: .test-infra/jenkins/job_beam_PreCommit_Website_Merge.groovy [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+  // Set common parameters.
+  common_job_properties.setTopLevelWebsiteJobProperties(delegate, 'mergebot')
+
+  triggers {
+    githubPush()
+  }
+
+  steps {
+    // Run the following shell script as a build step.
```
Comment (tgroh) [inline .test-infra/jenkins/job_beam_PreCommit_Website_Merge.groovy]: Undocumented shell stuff is suboptimal

Comment (tgroh) [inline .test-infra/jenkins/job_beam_PreCommit_Website_Merge.groovy]: More specifically, how are we getting the arguments to things like `gpg`? Magic constant arguments definitely don't enthuse me.

Comment (jasonkuster) [inline .test-infra/jenkins/job_beam_PreCommit_Website_Merge.groovy]: Instructions are documented here: https://rvm.io/rvm/install.
Will add a comment to that regard.

**Pull request closed without merging**